src/gallia/commands/discover/doip.py

```diff
@@ -2,6 +2,8 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+from __future__ import annotations
```

Comment:
is deprecated in cpython, can be removed

Reply:
Due to the changes with `DoIPConnection` it is no longer needed anyhow.